Sponsors.md

```diff
@@ -1,90 +1,91 @@
 
+### Catering Sponsor
 
-## Catering Sponsor
 At the end of every meetup we have a networking session that offers - perhaps - the greatest value of our meetups: getting to know other professionals who might be able to advance your career forward, help out with an annoying problem you’ve been dealing with, or, in time, even become your friends. 
 
-A sponsorship package of 100 EUR / meetup helps our meetup attendees satisfy their hunger and thirst, all in the good name of your company. 
-
+Get in contact with us to discuss catering opportunities.
 
 The list below describes the awareness benefits that our catering sponsor receives in return:
-* leaflets (can be left on table at entrance),
+* leaflets (can be left on a table at entrance),
 * online banner on DevStaff website (with back-link do-follow),
-* “Thank-you” message by organizers during Meetup (not by company),
-* Sponsor’s slide in DevStaff Welcome presentation,
-* “Thank you” mention in facebook page post.
+* "Thank you" message by organizers during Meetup (not by company),
+* Sponsor's slide in DevStaff Welcome presentation,
+* "Thank you" mention in our social media.
+
+### Venue Sponsor
 
-## Venue Sponsor
 High quality human interactions happen more naturally in a nice setting. The venue is one of the most important factors that go into running successful meet-ups. 
 
-We want venues that are accessible, large enough for our size and provide the necessary equipment (e.g. fast internet connection, projector, comfortable seating, etc.) to help our meet-ups run smoothly. You can find more details in [this relevant discussion](https://github.com/devstaff-crete/DevStaff-Heraklion/issues/185). 
+We want venues that are accessible, large enough for our size and provide the necessary equipment (e.g. fast internet connection, projector, comfortable seating, etc.) to help our meet-ups run smoothly. You can find more details in [this relevant discussion](https://github.com/devstaff-crete/DevStaff-Heraklion/issues/185). We prefer a venue that is free and without any unreasonable demands. 
 
 The list below describes the awareness benefits that our venue sponsor receives in return:
-* leaflets (can be left on table at entrance),
+* leaflets (can be left on a table at entrance),
 * online banner on DevStaff website (with back-link do-follow),
-* “Thank-you” message by organizers during Meetup (not by company),
+* "Thank-you" message by organizers during Meetup (not by company),
 * Sponsor’s slide in DevStaff Welcome presentation,
-* “Thank you” mention in facebook page post.
+* "Thank you" mention in our social media.
 
-## How to contact us about Sponsorship opportunities
+### How to contact us about Sponsorship opportunities
 Depending on the time of year and ongoing sponsor engagements, some, or all, of the above options may or may not be available. Please contact us at info @t devstaff d0t gr, so you can get in touch with the Org Team and discuss the available options. 
```

Comment:
Can't we replace `info @t devstaff d0t gr` with [info@devstaff.gr](mailto:info@devstaff.gr) ?